### PR TITLE
PG to PG: TSTZRange and JSONB

### PR DESCRIPTION
**Pull request by Amogh-Bharadwaj**
PG to PG CDC:
- Adds support for JSONB (now maps to JSONB on target)
- Constructs the string for tstzrange (still text on target) in a way that you can cast to tstzrange like select mytstzrange::tstzrange from table. This would fail before as the string we were inserting was of some json type string. Null lower and upper bounds tested

Functionally tested

### CODE REVIEW

**Review by iskakaushik — APPROVED** [review]

File: flow/connectors/postgres/qvalue_convert.go [hunk 1]
```diff
@@ -481,3 +512,23 @@ func customTypeToQKind(typeName string) qvalue.QValueKind {
 		return qvalue.QValueKindString
 	}
 }
+
+// Postgres does not like timestamps of the form 2006-01-02 15:04:05 +0000 UTC
+// in tstzrange.
+// ConvertTimeRangeBounds removes the +0000 UTC part
+func ConvertTimeRangeBounds(timeBound interface{}) (string, error) {
```
Comment (heavycrystal) [inline flow/connectors/postgres/qvalue_convert.go:519]: unexport

**Comment by serprex** [comment]
What's the motivation for this? PG<>PG should use PG type system to have this accuracy & more

File: flow/connectors/postgres/qvalue_convert.go [hunk 2]
```diff
@@ -108,6 +110,8 @@ func (c *PostgresConnector) postgresOIDToQValueKind(recvOID uint32) qvalue.QValu
 		return qvalue.QValueKindArrayString
 	case pgtype.IntervalOID:
 		return qvalue.QValueKindInterval
+	case pgtype.TstzrangeOID:
```
Comment (serprex) [inline flow/connectors/postgres/qvalue_convert.go:113]: should also handle tsrange at least

**Comment by serprex** [comment]
#1733 has merged, making this obsolete

**Pull request closed without merging**